src/ErlangTerm.jl

```diff
@@ -243,4 +284,13 @@ function deserialize(io, ::Val{STRING})
     array
 end
 
+function _serialize(io, ::Nothing)
+    write(io, NIL)
+    io
+end
```

Comment:
I do not think that this is correct. As far as I understand, `NIL` is an empty list (see `_serialize(io, array::AbstractArray)`) whereas `Nothing` should be the atom `:nil`.

```suggestion
function _serialize(io, ::Nothing)
    _serialize(io, :nil)
    io
end
```